src/context/schemas/chatInitSettings.schema.json

```diff
@@ -13,8 +13,15 @@
         "members": {
```

Comment:
missing `"allOf": [{ "$ref": "context.schema.json#" }],` above `porperties` to import the context schema and extend it

(Sorry couldn't put this on the correct line).